.github/workflows/integration_tests.yaml

```diff
@@ -1,4 +1,4 @@
-name: tests
+name: integration-tests
```

Comment:
sorry i should have done this! didnt think to split them up

Reply:
No problems! Things were still in flux with my open PR on aieng-template changes.